docker-compose.yml

```diff
@@ -50,4 +50,4 @@ services:
       - DB_USERNAME=postgres
       - DB_PASSWORD=developmentpassword
       - SETTINGS__APPLICATION=teacher-training-api-bg
-      - REDIS_URL=redis://redis:6379
+      - REDIS_CACHE_URL=redis://redis:6379
```

Comment:
Shouldn't it be the worker redis here? And probably above as well
REDIS_URL may have been wrong the whole time

Reply:
@darokel @defong Ref my previous comment here.

Reply:
Yes should be the worker one going forward